packages/vue/.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
```

Comment:
why aren't we using the linting rules from the root?

Reply:
because different plugins are needed for vue